src/main.js

```diff
@@ -1,6 +1,7 @@
 import process from "node:process";
 import { cases } from "./cases/index.js";
 import { logger } from "./common/logger.js";
+import "./common/proxy.js";
```

Comment:
Someone coming to this later may not realise that this line is actually doing anything.

Maybe wrap the proxy in a function and call it explicitly rather than rely on the side effect of the import?